src/types/onyx/SearchResults.ts

```diff
@@ -64,6 +64,17 @@ type SearchResultsInfo = {
 
     /** The optional columns that should be shown according to policy settings */
     columnsToShow: ColumnsToShow;
+
+    /** The status of the search results to show */
+    statusToShow: {
+        // eslint-disable-next-line jsdoc/require-jsdoc
+        expense: {
+            // eslint-disable-next-line jsdoc/require-jsdoc
+            drafts: boolean;
+            // eslint-disable-next-line jsdoc/require-jsdoc
+            approved: boolean;
```

Comment:
We actually return all statuses to the client, e.g. `all`, `drafts`, `outstanding`, `approved`, `paid`

Reply:
So should I add all the fields? I am not sure if I understand this part of the code

Reply:
Yea, I think we should add all of them for now since they are always returned by the server

Reply:
Do you mean these statuses? I don't see any `outstanding` in the costs.
```js
STATUS: {
  ALL: 'all',
  SHARED: 'shared',
  DRAFTS: 'drafts',
  FINISHED: 'finished',
},
```

Reply:
Not sure where to look for these values 😄

Reply:
They are [here](https://docs.google.com/document/d/1bRV1nPz8SaI_6DHqfehmLnADxv40us_q0JNvmZ1QyaY/edit#bookmark=id.d4v3gb8qe7kn). We should still support `Shared` for now, but we'll deprecate that one soon and add the other statuses mentioned in the doc.

Reply:
why do we have to add this in this PR? I don't understand why this suddenly appeared here, when I don't see this being used anywhere.

Can I just remove it if it's unused and we can add it in the PR that this is actually being used?